pom.xml

```diff
@@ -1,20 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
+  ~ Copyright 2018 ABSA Group Limited
```

Comment:
Shouldn't copyright be the topmost things, above the xml def?

Reply:
It was automatically added there.
If I move it to topmost position it says "Xml declaration should precede all document content"

Reply:
Let's believe the IDE